### PR TITLE
Ipr 3355 radio 2 new music playlist

### PR DESCRIPTION
**Pull request by radiocontrolled**
see https://jira.dev.bbc.co.uk/browse/IPR-3355

Added translation for:
"new_music_playlist"

In languages:

English - en
Cymraeg (Welsh) - cy - Rhestr chwarae cerddoriaeth newydd Radio 2
Gaeilge (Irish) - ga - Seinnliosta Ceoil Úir Radio 2
Gàidhlig (Scottish Gaelic) - gd - Clàr-ciùil ùr Radio 2

### CODE REVIEW

File: src/RMP/Translate/lang/programmes/en.po [hunk 1]
```diff
@@ -1249,6 +1249,11 @@ msgstr "National Radio"
 msgid "nations_radio"
 msgstr "Nations Radio"
 
+#. English: "Radio 2 New Music Playlist".
+#. Used by: Radio v1
+msgid "new_music_playlist"
+msgstr ""
```
Comment (craigharvi3) [inline src/RMP/Translate/lang/programmes/en.po]: Are you missing the English translation here @radiocontrolled ?

Comment (radiocontrolled) [inline src/RMP/Translate/lang/programmes/en.po]: 😂 I'll fix that now @craigharvi3

**Comment by steveculverwell** [comment]
:squid:

**Comment by djmarland** [comment]
Sorry I missed this earlier.
This would have been better to be parameterised: `%1 New Music Playlist`
What happens when 6 Music decide they need a Playlist? Does another one have to be added?

**Comment by radiocontrolled** [comment]
@djmarland I can update the translation but I did this in rmp-translate before realising it needed to be a forge translation (it's there now). 

The way Sion exlained the translation for me was that it wasn't straightforward word substitution and he suggested it work as a sentence, so I treated it as one-off.  With parameterisation can this take into account that `%1` needs to go after `New Music Playlist` for Scottish/Irish/Welsh?

**Comment by djmarland** [comment]
Indeed the parameterisation allows you to move the substituted word to whichever point in the sentence it is supposed to be at for that language. So yes the `%1` goes at the end for those languages.

**Comment by BPScott** [comment]
@radiocontrolled: An example of that which is already in place is the translation for `atoz_letters`. Note how the you are free to change the location of the placeholder when comparing [Arabic](https://github.com/bbc/rmp-translate/blob/40ac6c6a25789fc9e246b905ab953d937d80534f/src/RMP/Translate/lang/programmes/ar.po#L91-L93) vs [English](https://github.com/bbc/rmp-translate/blob/40ac6c6a25789fc9e246b905ab953d937d80534f/src/RMP/Translate/lang/programmes/en.po#L91-L93)

**Pull request merged**